README.md

```diff
@@ -26,41 +26,44 @@ sudo mv ./kubectl-argo-rollouts-linux-amd64 /usr/local/bin/kubectl-argo-rollouts
 
 ## Draft
 
-test argo rollout version
-
+Install Argo Rollouts
 ```
-kubectl argo rollouts version
+kubectl create namespace argo-rollouts
+kubectl apply -n argo-rollouts -f https://raw.githubusercontent.com/argoproj/argo-rollouts/stable/manifests/install.yaml
 ```
 
-have istioctl installed and install istio
+Install Argo CLI
+```
+curl -LO https://github.com/argoproj/argo-rollouts/releases/latest/download/kubectl-argo-rollouts-linux-amd64
+chmod +x ./kubectl-argo-rollouts-linux-amd64
+sudo mv ./kubectl-argo-rollouts-linux-amd64 /usr/local/bin/kubectl-argo-rollouts```
 
+test Argo Rollouts version
 ```
-istioctl install --set profile=demo
+kubectl argo rollouts version
 ```
 
+Install Prometheus and Grafana
+
 ```
-kubectl get all -n istio-system
+kubectl apply -f manifests/generated/monitoring -n demo   
 ```
 
-Install prometheus add on
-
+Open Prometheus dashboard
 ```
-kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.9/samples/addons/prometheus.yaml
+ kubectl port-forward -n demo service/prom 9090
```

Comment:
Do we need this? I used nodePort service with particular port so you should have access to this from your local machine (that works on katacoda at least)

Reply:
I don't think it works with KinD without port-forwarding